examples/k8s-upgrade/README.md

```diff
@@ -2,18 +2,116 @@
 
 ## Overview
 
-This document describes how to upgrade kubernetes version on a running cluster.
+This document describes how to upgrade kubernetes version for an existing cluster.
 
-Supported scenarios:
-- upgrade from v1.5.x to the latest supported version in v1.6 stream
-- upgrade from v1.6.x to the latest supported version in v1.7 stream
-- upgrade from v1.7.x to the latest supported version in v1.8 stream
+*acs-engine* supports Kubernetes version upgrades starting from 1.5 release.
+During the upgrade, *acs-engine* successively visits virtual machines that constitute the cluster and performs the following operations:
+ - cordon the node and drain existing workload
+ - delete the VM
+ - create new VM and install desired orchestrator version
+ - add the new VM to the cluster
 
-The cluster definition file examples demonstrate initial cluster configurations:
-- **kubernetes1.5.json** - Kubernetes cluster v1.5 with Linux agent pool
-- **kubernetes1.5-win.json** - Kubernetes cluster v1.5 with Windows agent pool
-- **kubernetes1.5-hybrid.json** - Kubernetes cluster v1.5 with Linux and Windows agent pools
-- **kubernetes1.6.json** - Kubernetes cluster v1.6 with Linux agent pool
-- **kubernetes1.7.json** - Kubernetes cluster v1.7 with Linux agent pool
+*acs-engine* allows a subsequent minor version upgrade at a time, for example, from 1.5.x to 1.6.x.

```

Comment:
substitute "a subsequent minor version" with "one subsequent minor version"

Reply:
Do you think it's useful or not to explain how master and agent nodes got upgrade?

Reply:
@weinong lines 8-12 don't do that?